mysite/mysite/tests/test_authentication.py

```diff
@@ -0,0 +1,144 @@
+import json
+
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+from django.conf.urls import url
+from django.core.urlresolvers import reverse
+from django.test import RequestFactory, TestCase
+
+from jmbo.models import ModelBase
+
+from mysite.tests.urls import urlpatterns
+
+from rest_framework import generics
```

Comment:
Move this block up to below the django imports please. They're higher up in the foodchain.